content/en/_index.md

```diff
@@ -49,32 +48,37 @@ OpenTelemetry is a collection of APIs, SDKs, and tools. Use it to instrument,
 generate, collect, and export telemetry data (metrics, logs, and traces) to help
 you analyze your software's performance and behavior.
 
-> OpenTelemetry is **generally available** across
-> [several languages](docs/languages/) and is suitable for use.
+> OpenTelemetry is [generally available](/status/) across
+> [several languages](docs/languages/) and is suitable for production use.
 
 {{% /blocks/lead %}}
 
 {{% blocks/section color="dark" type="row" %}}
 
-{{% blocks/feature icon="fas fa-chart-line" title="Traces, Metrics, Logs"%}}
+{{% blocks/feature icon="fas fa-chart-line" title="Traces, Metrics, Logs" url="docs/concepts/observability-primer/" %}}
 
 Create and collect telemetry data from your services and software, then forward
```

Comment:
```suggestion
Create and collect telemetry from your services and software, then forward
```

Reply:
Yes, I had a similar thought to drop "data" 👍🏻